src/Domain.jl

```diff
@@ -480,7 +480,7 @@ function ConstantTVDomain(;phase::Z,initialconds::Dict{X,E},constantspecies::Arr
         if key == "T"
             T = val
         elseif key == "P"
-            P = val
+            throw(error("ConstantTVDomain cannot specify P"))
```

Comment:
Should we do this in ParameterizedPConstantVDomain as well?

Reply:
This Domain already has an input check for P in the later part. I moved the checking code a bit forward.